Assets/MixedRealityToolkit/_Core/Managers/MixedRealityManager.cs

```diff
@@ -979,5 +994,30 @@ private static bool CheckComponentMatch(Type type, string managerName, Tuple<Typ
         #endregion Manager Utilities
 
         #endregion Manager Container Management
+
+        #region Manager Accessors
+
+        private static IMixedRealityInputSystem inputSystem = null;
```

Comment:
I'm usually against Statics but this would save time.

However, this will ONLY work for the Core managers, components / devices and runtime managers will still need to use the discovery system

Reply:
Yeah I only intended to put core managers here.

I already do something similar in the input profile to get dictation and speech manatees l managers using the discovery system.